dev-libs/libprelude/libprelude-3.0.0.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{3,4,5}} )
+
+USE_RUBY="ruby21"
+
+inherit autotools eutils python-r1 ruby-single
+
+DESCRIPTION="Prelude-SIEM Framework Library"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc lua python ruby perl"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="dev-libs/libgcrypt:0=
+	net-libs/gnutls
+	perl? ( dev-lang/perl virtual/perl-ExtUtils-MakeMaker )
+	lua? ( dev-lang/lua:* )
+	ruby? ( ${RUBY_DEPS} )
+	python? ( ${PYTHON_DEPS} )"
+
+DEPEND="${RDEPEND}
+	doc? ( dev-util/gtk-doc )
+	sys-devel/flex
+	virtual/yacc
+	>=dev-lang/swig-3.0.7
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}/${P}-perl-hardening.patch"
+	"${FILESDIR}/${P}-ruby-m4.patch"
+	"${FILESDIR}/${P}-ruby-configure.patch"
+)
+
+src_prepare() {
+	default_src_prepare
+
+	# Avoid null runpaths in Perl bindings.
+	sed -e 's/ LD_RUN_PATH=""//' -i "${WORKDIR}/${PF}/bindings/Makefile.am" || die "sed failed"
+
+	mv "${WORKDIR}/${PF}/configure.in" "${WORKDIR}/${PF}/configure.ac" || die "mv failed"
+
+	eautoreconf
+}
+
+src_configure() {
+	python2_configure=--without-python2
+	python3_configure=--without-python3
```

Comment:
Please declare these local since they are irrelevant outside the phase function.